inference-engine/tests/functional/plugin/cpu/bfloat16/bfloat16_helpers.hpp

```diff
@@ -179,7 +179,7 @@ class BasicBF16Test : public testing::WithParamInterface<basicParams>,
     }
 
     void test() {
-        if (!InferenceEngine::with_cpu_x86_bfloat16()) {
+        if (!InferenceEngine::with_cpu_x86_avx512_core()) {
```

Comment:
Looks like it would be nice to extend the comment below that on platforms with avx512 isa but w/o native bfloat16 support computations are done via simulation mode.

Reply:
Done